kotest-framework/kotest-framework-multiplatform-plugin-gradle/test-project/build.gradle.kts

```diff
@@ -68,3 +76,17 @@ if (useNewNativeMemoryModel.toBoolean()) {
       }
    }
 }
+
+// Use a Node.js version current enough to support Kotlin/Wasm
+
+rootProject.plugins.withType<org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootPlugin> {
+   rootProject.extensions.configure<org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootExtension> {
+      nodeVersion = "22.0.0-v8-canary20231213fc7703246e"
```

Comment:
Should this be updated/changed before merge?

Reply:
The regular 22.x Node.js release is scheduled for April. Until then, we must carefully select a pre-release which works with the Kotlin compiler infra (newest 22.0.0 canaries no longer support the '--experimental-wasm-gc' option). Meanwhile, I'll update to a compatible Node.js release mentioned in KT-65864.